app/views/form/check_answers.html.erb

```diff
@@ -22,7 +22,13 @@
       </dl>
 
       <%= form_with model: @case_log, method: "get" do |f| %>
-        <%= f.govuk_submit "Save and continue"  %>
+        <%= f.govuk_submit 'Save and continue' do %>
+          <% if @case_log.status == "in_progress" && @case_log.status == "completed" || @case_log.form.all_subsections_except_declaration_completed?(@case_log) == false %>
+            <a class="govuk-button govuk-button--secondary" href=<%= "/logs/#{@case_log.id}/#{@case_log.form.next_incomplete_section_redirect_path(subsection, @case_log)}" %>>Save and go to next incomplete section</a>
```

Comment:
Can use `govuk_link` helper for these links. See https://govuk-components.netlify.app/helpers/button/#link-styled-like-a-button

Reply:
Thanks for the tip, I originally got the above from https://govuk-form-builder.netlify.app/form-elements/submit/#a-group-of-buttons do you think it might be worth suggesting that they update the page to reference the helpers too or link out to the helpers?

Reply:
Ah, interesting, and actually makes me realise… shouldn’t be buttons, not links, if they submit a form? What happens if JS is unavailable?

Otherwise, maybe it’s worth a PR on the GOV.UK Form Builder repo, asking if pages should use or mention components from the GOV.UK Components library.

Reply:
I can answer that! 😄 under the hood these links/buttons on the check answers page aren't actually submitting anything. By the time you reach the check answers page you've already submitted all the questions you've answered since we're actually submitting after each question. These buttons/links just act as redirects to elsewhere in the case log process without performing any kind of submission so theoretically they shouldn't be affected by JS being unavailable (might be worth @baarkerlounger confirming this)

Reply:
Yeah they are links rather than real buttons. Don't think they should be affected by JS at all but, as always, worth testing.